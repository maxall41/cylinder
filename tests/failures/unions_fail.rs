use embedded_cylinder::CylinerBuildU8;




#[derive(CylinerBuildU8)]
union XYZ {
    cubes: u8,
    eggs: u16
}