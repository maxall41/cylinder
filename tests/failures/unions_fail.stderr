error: Can not run on Unions! Cylinder only works on enums
 --> tests/failures/unions_fail.rs:6:10
  |
6 | #[derive(CylinerBuildU8)]
  |          ^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `CylinerBuildU8` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0601]: `main` function not found in crate `$CRATE`
  --> tests/failures/unions_fail.rs:10:2
   |
10 | }
   |  ^ consider adding a `main` function to `$DIR/tests/failures/unions_fail.rs`
